INN/umbrella-sfpublicpress
# Add promo box widget

## Changes

This pull request makes the following changes:

- Adds a new `San Francisco Public Press Promo Box` widget and styling for it (image uploader portion is stolen from the Largo Image widget)

Desktop:
![Screen Shot 2020-01-13 at 2 49 39 PM](https://user-images.githubusercontent.com/18353636/72286849-fae7df80-3613-11ea-859b-5941b060d36d.png)

Mobile:
![Screen Shot 2020-01-13 at 2 49 45 PM](https://user-images.githubusercontent.com/18353636/72286848-fae7df80-3613-11ea-88e5-2b2d338fcaf3.png)

Reverse background:
![Screen Shot 2020-01-13 at 2 49 30 PM](https://user-images.githubusercontent.com/18353636/72286850-fae7df80-3613-11ea-8cb4-0ca51f397a1b.png)

## Why



For #18

## Testing/Questions

Features that this PR affects:

- Homepage promo box widget


Questions that need to be answered before merging:

- [ ] Does the widget image uploader work ok?
- [ ] Did I miss anything that should've been removed while revamping this widgets code?

Steps to test this PR:

1. Add the new `San Francisco Public Press Promo Box` widget to the homepage bottom widget area
2. Give it a title, text, image, and image link
3. View it on desktop and mobile
4. Switch to the `reverse` background and make sure it works ok
